docs/CLI_OUTPUT.md

```
@@ -0,0 +1,171 @@
+tsumugai play scenario.md --debug    # デバッグ情報付き
+```
+
+> `trace` / `dry-run` コマンドは将来追加予定（Issue #38, #40）。
+
+---
+
+## check コマンド：人間向け出力
+
+### 問題なし（正常）
+
+```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)